Editor/Resources/EditorWindow/Components/ConnectToFleetInput.uxml

```diff
@@ -0,0 +1,40 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <ui:Style src="./common.uss"/>
+    <ui:VisualElement name="AnywherePageCreateFleet" class="separator">
+        <ui:VisualElement name="AnywherePageFleetName" class="form-row">
```

Comment:
Please rename so names contain suffix same as the parent component - so maybe `AnywherePageCreateFleet<ComponentType>`? Same for the next line.

Reply:
Updated